frontend_v2/src/app/components/challenge/challengeleaderboard/challengeleaderboard.component.html

```diff
@@ -87,14 +90,15 @@ <h5 class="fw-light">Leaderboard</h5>
                 </div>
                 <span class="w-400"> {{ sortLeaderboardTextOption }}</span>
               </mat-chip-list> -->
-              <mat-checkbox
-                class="pull-right complete-leaderboard"
-                *ngIf="isChallengeHost && showLeaderboardToggle"
-                (change)="toggleLeaderboard(getAllEntries)"
-                [(ngModel)]="getAllEntries"
-              >
-                <label class="fw-light complete-leaderboard-text"> {{ getAllEntriesTextOption }}</label>
-              </mat-checkbox>
+              </div>
+                <mat-checkbox
```

Comment:
@gautamjajoo  This checkbox will appear if we have total submission `0` We need a check here that only if my total submission(both public and private) is >0, then we should show this checkbox

Reply:
Fixed. Now the checkbox only appears when `public + private > 0`.